core/runtime/tests/snapshot.rs

```diff
@@ -1,4 +1,5 @@
 // Copyright 2018-2023 the Deno authors. All rights reserved. MIT license.
+use crate as deno_core;
```

Comment:
I might remove the `op2(core)` and just do this in the rare files in deno_core that have ops.

Reply:
I like this idea

Reply:
I'll follow up with it in a later pass!